Source/Forms/Gen4/Wild4.cpp

```diff
@@ -296,12 +297,32 @@ void Wild4::search()
     searcherModel->clearModel();
     searcherModel->setMethod(method);
 
-    ui->pushButtonSearch->setEnabled(false);
-    ui->pushButtonCancel->setEnabled(true);
-
     std::array<u8, 6> min = ui->filterSearcher->getMinIVs();
     std::array<u8, 6> max = ui->filterSearcher->getMaxIVs();
 
+    auto encounter = static_cast<Encounter>(ui->comboBoxSearcherEncounter->currentData().toInt());
+    if (encounter == Encounter::BugCatchingContest)
+    {
+        bool flag = true;
+        for (u8 i = 0; i < 6; i++)
+        {
+            if (min[i] == 31)
+            {
+                flag = false;
+                break;
+            }
+        }
+        if (flag)
+        {
+            QMessageBox err(QMessageBox::Warning, ("Missing Flawless IV"), ("BCC Searcher needs at least one IV at 31"));
```

Comment:
Wrap the strings in tr() calls. Also use "Bug Contest" instead of BCC

Reply:
done